src/utils/ValidationHelper.ts

```diff
@@ -28,4 +33,59 @@ export class ValidationHelper {
 		});
 	}
 
+	private validateUserData(data: string | undefined, errmsg: string, logger: Logger): boolean {
+		let isValid = true;
+		if (data == null || data === undefined || data.trim().length === 0) {
+			logger.info("UserInfo missing: ", errmsg);
+			isValid = false;
+		}
+		return isValid;
+	}
+
+	validateUserInfo(session: ISessionItem, logger: Logger): boolean {
+		let isValid = true;
+		if (!this.validateUserData(session.full_name, "Full Name is missing", logger) ||
+			!this.validateUserData(session.date_of_birth, "Date of Birth is missing", logger) ||
+			!this.validateUserData(session.document_selected, "Document selection type is missing", logger) ||
+			!this.validateUserData(session.date_of_expiry, "Expiry Date is missing", logger)) {
+			isValid = false;
+		}
+		return isValid;
+	}
+
+	async eventToSubjectIdentifier(jwtAdapter: KmsJwtAdapter, event: APIGatewayProxyEvent): Promise<string> {
+		const headerValue = event.headers.authorization ?? event.headers.Authorization;
+		if (headerValue === null || headerValue === undefined) {
+			throw new AppError( "Missing header: Authorization header value is missing or invalid auth_scheme", HttpCodesEnum.UNAUTHORIZED);
+		}
+		const authHeader = event.headers.Authorization as string;
+		if (authHeader !== null && !authHeader.includes(Constants.BEARER)) {
+			throw new AppError( "Missing header: Authorization header is not of Bearer type access_token", HttpCodesEnum.UNAUTHORIZED);
+
+		}
+		const token = headerValue.replace(/^Bearer\s+/, "");
```

Comment:
will that Constants work here

Reply:
I have tried extracting it to a const but doesnt work.